src/components/UrlError/urlError.jsx

```diff
@@ -0,0 +1,18 @@
+import './urlError.css'
+import { Link } from 'react-router-dom'
+
+export default function UrlError() {
+  return (
+    <div className='url-error'>
+      <Link to={`/`}>
```

Comment:
In future iterations, it would be interesting to see how we could DRY up the use of this return to home button by conditionally rendering the UrlError component from the SingleMovie component.